tools.py

```diff
@@ -676,6 +705,11 @@ def find_solution_for_frontend_input_output_shapes_divisibility(
 	cmd.add_argument('--output-path', '-o', default = 'data')
 	cmd.add_argument('--comment-path', '-c')
 	cmd.set_defaults(func = processcomments)
+
+	cmd = subparsers.add_parser('cleanup_transcripts_with_empty_ref')
+	cmd.add_argument('--input-path', '-i', required = True)
+	cmd.add_argument('--emptiness', default = 1)
```

Comment:
Please rename to `min_ref_length` (and maybe change the default into 2)

Reply:
done